_toc.yml

```diff
@@ -11,6 +11,7 @@ parts:
   - file: examples/nist-example.md
   - file: examples/NREL-example.md
   - file: examples/pnnl-example1.md
+  - file: examples/pd-sr-mp-pritoni.md
```

Comment:
Is there a better name for this file? Maybe just `pritoni.md`?

Reply:
We could, but I didn't want to step on the toes of others who also modeled the Pritoni example using bob. We could discuss on Friday...

Reply:
Sounds good. Seems like there should be a single "canonical" Pritoni model for our and other's purposes. 

We could also consider putting this model in the `Example Systems` section here.